### PR TITLE
Fix fit sampling

### PR DESCRIPTION
**Pull request by brownd1978**
Make standard tracker positions fit sampling explicitly when extrapolating. Also some minor cleanup.

### CODE REVIEW

**Comment by brownd1978** [comment]
@FNALbuild run build test

File: Mu2eKinKal/fcl/prolog.fcl [hunk 1]
```diff
@@ -396,7 +395,7 @@ Mu2eKinKal : {
     MaterialSettings : @local::Mu2eKinKal.MAT
     KKFitSettings: {
       @table::Mu2eKinKal.KKFIT
-      SampleSurfaces : ["TT_Front","TT_Mid","TT_Back","ST_Outer","ST_Back","OPA","TSDA"]
+      SampleSurfaces : ["ST_Outer","ST_Back"] # these are additional surfaces; surfaces used in extrapolation are also sampled
```
Comment (kutschke) [inline Mu2eKinKal/fcl/prolog.fcl:398]: Are the three removed surfaces implicitly present or are they removed.  If implicit please update the comment to say so.

**Review by kutschke — APPROVED** [review]
One minor comment about comments.  I will watch for Richie's approval.

File: Mu2eKinKal/fcl/prolog.fcl (hunk 1 above)
Comment (brownd1978) [reply Mu2eKinKal/fcl/prolog.fcl:398]: The surfaces are explicit in the extrapolation configuration. These additional surfaces will all go away once the extrapolation codes are complete.

**Comment by brownd1978** [comment]
@FNALbuild run build test

**Pull request merged**